primitives/core/src/lib.rs

```diff
@@ -81,6 +80,8 @@ pub use self::hasher::blake2::Blake2Hasher;
 pub use self::hasher::keccak::KeccakHasher;
 pub use hash_db::Hasher;
 
+pub use bounded_collections as bounded;
+pub use bounded_collections::*;
```

Comment:
Before `BoundedVec` etc also wasn't exported in the top level of the crate. We should probably change this again?

Reply:
A couple of types were in fact exported from the top level, and those were the types that aren't strictly related to the bounded types, e.g. `TypedGet`, `Get`, `TryCollect`, etc. Should I export only these types from the top level then?

Reply:
I would say yes.